src/sidebars/sidebar.less

```diff
@@ -35,6 +35,16 @@
   -webkit-animation-name: favourites;
 }
 
+.version {
+  bottom: 15px;
+  left: 70px;
+  position: absolute;
+  color: @icon-colour;
+  font-weight: @font-light;
+  font-size: 11px;
+  cursor: default;
+}
```

Comment:
I'm not sure, it's just a small number in a corner of the page, but I'm happy to do so